src/database/__init__.py

```
@@ -1,20 +1,41 @@
-import os
-
-from database.models import (
+from .models import (
     Base,
-    MovieModel
+    MovieModel,
+    GenreModel,
+    ActorModel,
+    CountryModel,
+    LanguageModel,
+)
+
+from .session_sqlite import (
+    init_db,
+    close_db,
+    get_db,
+    get_sqlite_db_contextmanager as get_db_contextmanager,
+    reset_sqlite_database,
+    reset_database,
 )
-from database.session_sqlite import reset_sqlite_database as reset_database
 
-environment = os.getenv("ENVIRONMENT", "developing")
+try:
+    from .populate import CSVDatabaseSeeder  # noqa: F401
+except Exception:
+    CSVDatabaseSeeder = None  # optional
```

The try/except around importing CSVDatabaseSeeder is too broad and will silence any exception raised during import (including syntax errors or other bugs). This makes debugging harder and can hide real problems. Prefer catching only ImportError and consider logging a warning when the seeder isn't available. Example: `except ImportError: CSVDatabaseSeeder = None`.